apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -171,7 +172,7 @@ const BookingPage = ({ eventType, booking, profile }: BookingPageProps) => {
     return {
       name: primaryAttendee.name || "",
       email: primaryAttendee.email || "",
-      guests: booking.attendees.slice(1).map((attendee) => attendee.email),
+      guests: !isDynamicGroupBooking ? booking.attendees.slice(1).map((attendee) => attendee.email) : [],
```

Comment:
Currently, the way guests are pulled in reschedule, all but the primary attendee are added as guests and this becomes a problem when dealing with Dynamic Group booking because the group itself contains all the Organizers but the reschedule logic adds ALL but primary organizer as guests meaning they all would get emails twice, thrice and so on as many times as the event is rescheduled. For this reason, I've placed this check here to disable guests temporarily for dynamic bookings, until a follow up takes care of this.